lib/core/src/lib/viewer/components/viewer-render/viewer-render.component.spec.ts

```diff
@@ -476,4 +476,73 @@ describe('ViewerComponent', () => {
             });
         });
     });
+
+    describe('Spinner', () => {
+        it('should show spinner when isLoading is true', () => {
+            component.isLoading = true;
+            fixture.detectChanges();
+            expect(testingUtils.getByCSS('.adf-viewer-render-main-loader')).not.toBeNull();
+        });
+
+        it('should show spinner until content is ready when viewerType is media', () => {
+            component.isLoading = false;
+            component.urlFile = 'some-file.mp4';
+
+            component.ngOnChanges();
+            fixture.detectChanges();
+
+            expect(testingUtils.getByCSS('.adf-viewer-render-main-loader')).not.toBeNull();
+
+            const mediaViewer = testingUtils.getByCSS('adf-media-player');
```

Comment:
instead checking component by tag's name you can use getByDirective function from testingUtils and then get this element by class name (MediaPlayerComponent)

Reply:
good point, applied